service/history/workflow/context.go

```diff
@@ -191,35 +184,19 @@ func NewContext(
 		throttledLogger: throttledLogger,
 		metricsHandler:  metricsHandler.WithTags(metrics.OperationTag(metrics.WorkflowContextScope)),
 		config:          config,
-		mutex:           locks.NewPriorityMutex(),
```

Comment:
Do you plan to remove the code for PriorityMutex?

Reply:
Yes, we can remove that now.